automated install/basic-install.sh

```diff
@@ -102,6 +105,40 @@ fi
 r=20
 c=70
 
+# Content of Pi-hole's meta package control file on APT based systems
+PIHOLE_META_PACKAGE_CONTROL_APT=$(
+    cat <<EOM
+Package: pihole-meta
+Version: 0.1
+Maintainer: Pi-hole team <adblock@pi-hole.net>
+Architecture: all
+Description: Pi-hole dependency meta package
+Depends: grep,dnsutils,binutils,git,iproute2,dialog,ca-certificates,cron,curl,iputils-ping,psmisc,sudo,unzip,libcap2-bin,dns-root-data,libcap2,netcat-openbsd,procps,jq,lshw,bash-completion
+EOM
+)
```

Comment:
Multi-line strings are also possible, for simplicity:
```suggestion
PIHOLE_META_PACKAGE_CONTROL_APT='Package: pihole-meta
Version: 0.1
Maintainer: Pi-hole team <adblock@pi-hole.net>
Architecture: all
Description: Pi-hole dependency meta package
Depends: grep,dnsutils,binutils,git,iproute2,dialog,ca-certificates,cron,curl,iputils-ping,psmisc,sudo,unzip,libcap2-bin,dns-root-data,libcap2,netcat-openbsd,procps,jq,lshw,bash-completion'
```

Reply:
Here Doc seems fine

Reply:
Sure it works, I was just wondering why to use a command constitution + `cat` (external command) + here document, when a simple string alone works as well.